facade/nodeFacade_test.go

```diff
@@ -1312,3 +1312,49 @@ func TestNodeFacade_GetTransactionsPool(t *testing.T) {
 		require.Equal(t, expectedPool, res)
 	})
 }
+
+func TestGetGasConfigs(t *testing.T) {
+	t.Parallel()
+
+	t.Run("empty gas configs map", func(t *testing.T) {
+		t.Parallel()
+
+		arg := createMockArguments()
+
+		wasCalled := false
+		arg.ApiResolver = &mock.ApiResolverStub{
+			GetGasConfigsCalled: func() map[string]map[string]uint64 {
+				wasCalled = true
+				return make(map[string]map[string]uint64)
+			},
+		}
+
+		nf, _ := NewNodeFacade(arg)
+		_, err := nf.GetGasConfigs()
+		require.Equal(t, ErrEmptyGasConfigs, err)
+		require.True(t, wasCalled)
+	})
+
+	t.Run("should work", func(t *testing.T) {
+		t.Parallel()
+
+		arg := createMockArguments()
+
+		wasCalled := false
+		arg.ApiResolver = &mock.ApiResolverStub{
+			GetGasConfigsCalled: func() map[string]map[string]uint64 {
+				wasCalled = true
+				return map[string]map[string]uint64{
```

Comment:
I would have extracted this map outside as `providedMap` and then compare the response on `GetGasConfigs` to be the provided one

Reply:
added additional check :+1: